exs/mundo_2/ruby/043.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Desafio 043
+
+Problema: Desenvolva uma lógica que leia o peso e a altura de
+          uma pessoa, calcule seu Índice de Massa Corporal (IMC)
+          e mostre seu status, de acordo com a tabela abaixo:
+            - IMC abaixo de 18,5: Abaixo do Peso
+            - Entre 18,5 e 25: Peso Ideal
+            - 25 até 30: Sobrepeso
+            - 30 até 40: Obesidade
+            - Acima de 40: Obesidade Mórbida
+
+Resolução do problema:
+=end
+
+print"Digite seu peso: "
+peso = gets.chomp.to_f
+print"Digite sua altura: "
+altura = gets.chomp.to_f
+
+imc = peso/(altura**2)
+
+puts"Seu imc é %.2f." % imc
+if 0 < imc and imc < 1.85
+	puts"Abaixo do peso."
+elsif 1.85 <= imc and imc < 25
```

Comment:
Corrija o erro de digitação: O valor deve ser 18,5; não 1,85
```suggestion
if 0 < imc and imc < 18.5
	puts"Abaixo do peso."
elsif 18.5 <= imc and imc < 25
```